src/components/bottomSheet/bottom-sheet.scss

```diff
@@ -132,7 +132,10 @@ md-bottom-sheet {
         @include grid-items-per-row(7);
       }
 
-
+      // Override of the IE11 fix from @mixin ie11-min-height-flexbug, line 109 mixins.scss
```

Comment:
As we discussed on Slack:

```js
// We have to remove the `pseudo` fix for IE on the list-item here, because we're 
// manually changing the `flex` layout of the list-item.
```

Can you maybe add something like that to the comment (to clarify the change - please do not use my weird sample :smile: )